034-kafka-topic-replication-factor.md

```diff
@@ -0,0 +1,76 @@
+# Support Replication Factor Change for Kafka Topics
+
+Apache Kafka introduced the Replication Factor (RF) as the configuration that determines the number replicas kafka brokers will retain for a single data partition. A bigger number of the RF means both higher reliability for the data partition and higher infrastructure cost for topics.
+
+Apache Kafka allows users to change the RF for any topic partition through the native scripts during the runtime. However, it's not a lightweight process, because users usually need to decide which brokers to host new replicas and which replicas need to be dropped, or even conduct a cluster rebalance after the change of the RF.  
+
+The goal of this proposal is to offer users a simple approach to change the topic RF through the KafkaTopic Custom Resources and Cruise Control, in the cloud native, strimzi style. 
+
+## Current situation
+
+Currently, users have two options to change the topic RF:
+* Leverage the Native Kafka script to change the RF. Users need to generate the partition replica assignment and balance the kafka clusters manually if needed. 
+* Delete the topic and recreate the topic with the desired RF.
+
+Clearly none of the above approaches is user-friendly.
```

Comment:
I would say that the second option is not about to be "user-friendly", it's not acceptable because of losing data, unless the user accepts that.